src/main/java/oshi/hardware/platform/windows/WindowsGlobalMemory.java

```diff
@@ -68,6 +69,18 @@ public long getTotal() {
     }
 
     @Override
+    public long getSwapTotal() {
+        // TODO: for now, it returns 0
+        return 0;
+    }
+
+    @Override
+    public long getSwapAvailable() {

```

Comment:
Amount used from Performance Monitor counters (see `WindowsCentralProcessor`) under "\Paging File(_Total)\% Usage".  Will have to reference that percent used to total and subtract used from total to get available.